contrib/dcr_tmux_simnet_setup.sh

```diff
@@ -25,12 +25,19 @@
 set -e
 
 SESSION="dcrd-simnet-nodes"
-NODES_ROOT=~/dcrdsimnetnodes
+NODES_ROOT=${DCR_SIMNET_ROOT:-${HOME}/dcrdsimnetnodes}
 RPCUSER="USER"
 RPCPASS="PASS"
 WALLET_SEED="b280922d2cffda44648346412c5ec97f429938105003730414f10b01e1402eac"
 WALLET_MINING_ADDR="SspUvSyDGSzvPz2NfdZ5LW15uq6rmuGZyhL" # NOTE: This must be changed if the seed is changed.
 WALLET_XFER_ADDR="SsonWQK6xkLSYm7VttCddHWkWWhETFMdR4Y" # same as above
+WALLET_CREATE_CONFIG=$(cat <<EOF
```

Comment:
shouldn't need a subshell for this, you could define a multiline string instead.

```
WALLET_CREATE_CONFIG="y
n
y
${WALLET_SEED}
"
```

whether you think that's more readable or not (because of the first y not at the start of the line is another matter though.

Reply:
I had it this way at first, but it wasn't working due to the same error above.  I tried it again now with the other change and it works fine, so I'll change this one back to just be the string.